recipes/panX/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "panx" %}
+{% set version = "0.0.0" %}
```

Comment:
Could you ask the authors to tag a release?

Reply:
We already asked them. ping @wdingx